server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlService.java

```diff
@@ -101,4 +123,19 @@ public List<AdmissionController> getAdmissionControllers() {
     public AdmissionController getAdmissionController(String controllerName) {
         return this.ADMISSION_CONTROLLERS.getOrDefault(controllerName, null);
     }
+
+    /**
+     * Return admission control stats
+     */
+    public AdmissionControlStats stats() {
+        List<AdmissionControllerStats> statsList = new ArrayList<>();
+        if (this.ADMISSION_CONTROLLERS.size() > 0) {
```

Comment:
nit -> camel pattern for private final or else make it static ?

Reply:
Will modify to camel pattern.